drivers/misc/rwbuffer.c

```diff
@@ -1182,6 +1182,8 @@ ssize_t rwb_readbytes(FAR struct rwbuffer_s *dev, off_t offset,
 #ifdef CONFIG_DRVR_REMOVABLE
 int rwb_mediaremoved(FAR struct rwbuffer_s *rwb)
 {
+  int ret = OK;
```

Comment:
```suggestion
  int ret;
```

Reply:
Styling in this file applies both therefore was not sure which way to apply.  e.g. refer to rwb_write().  If required can change to style of function below as opposed to function above.

Reply:
rwb_write() is different as it contains code that can lead to all branch installations evaluated to false or places that do not check ret < 0. In this case ret < 0 is always checked.